scripts/upgradeChecks.js

```diff
@@ -0,0 +1,140 @@
+const config = require('../config.json');
+let network = 'mainnet'
+
+async function main() {
+    const addr = config.addresses;
+
+    const netinfo = await ethers.provider.getNetwork();
+    network = netinfo.name;
+    if (network === "unknown")
```

Comment:
Branch this out into an `if......else` statement and deploy the contracts selectively based on the network arg.